infra/net/variables.tf

```diff
@@ -0,0 +1,79 @@
+variable "scale_period_in_seconds" {
+   type = string
+   default = 60
+}
+
+variable "scale_evaluation_periods" {
+   type = string
+   default = 2
+}
+
+variable "ami"{
+ type = string
+  default = "ami-05d47ff02376b216e"
+}
+
+variable "keyname"{
+  default = "bmaas"
+}
+
+variable "region" {
+  type        = string
+  default     = "eu-central-1"
+  description = "default region"
+}
+
+variable "vpc_cidr" {
+  type        = string
+  default     = "172.16.0.0/16"
+  description = "default vpc_cidr_block"
+}
+
+variable "pub_sub1_cidr_block"{
+   type        = string
+   default     = "172.16.1.0/24"
+}
+variable "pub_sub2_cidr_block"{
+   type        = string
+   default     = "172.16.2.0/24"
+}
+variable "prv_sub1_cidr_block"{
+   type        = string
+   default     = "172.16.3.0/24"
+}
+variable "prv_sub2_cidr_block"{
+   type        = string
+   default     = "172.16.4.0/24"
+}
+
+
+variable "sg_name"{
+ type = string
+ default = "alb_sg"
+}
+
+variable "sg_description"{
+ type = string
+ default = "SG for application load balancer"
```

Comment:
Not sure what SG means. If it's an acronym, please use the full version for extra clarity.

Reply:
SG - security group in AWS. The acronym is quite used this way, but we can change it.

Reply:
That's fine if it's just my lack of experience meaning I don't know it.